clodius/tiles/cooler.py

```diff
@@ -516,6 +502,8 @@ def make_mats(filepath):
 
         mats[filepath] = [f, info]
 
+    info['mirror_tiles'] = 'false'
```

Comment:
This is the one line that is neither a coordinate swap nor a whitespace tweak... It is what you intend?

Reply:
Nope. I forced no mirroring for debugging purposes. Good catch!

Reply:
The problem now is that the client performs "mirroring" by copying lower triangle pixels to the upper triangle, so the result is a white heatmap with pixels painted only on the main diagonal.